package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "todo_nodejs",
+  "version": "1.0.0",
+  "description": "Simple todo application in node js",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Please add the scripts to `start` the application like  ` "start":node index.js`

Reply:
please workk on this